pep/app/pods/components/page/sidebar/widgets/whats-new/template.hbs

```diff
@@ -2,11 +2,16 @@
     @scrollableNamespace="sidebar-right"
     @isOpen={{this.isOpen}}
     @toggle={{fn @toggleIsOpen this.widget}}
-    class="mb-2"
+    class="mb-2 page-widget"
     {{did-insert this.onElementInsert}}
     ...attributes
 as |Panel|>
-    <Panel.header @title={{t "relatedInfo.widgets.whatsNew.title"}} />
+    <Panel.header>
+        {{t "relatedInfo.widgets.whatsNew.title"}}
+        <span class="widget-icon" {{on "click" this.openModal}}>
+            <FaIcon @icon="envelope" @prefix="fal"/>
+        </span>
```

Comment:
![scared](https://i.imgur.com/FRI2Sko.gif)

Reply:
my list of concerns with this pattern is still unresolved 😛, but in the interest in making progress and getting this stuff merged in, we can punt on that. as i have time tho, i do intend to push a branch w/my proposed pattern of either my original design, or at least rework this new pattern a bit to alleviate some of my concerns.

Reply:
I added the items you wanted per your screenshot. I kept the icons as well to make it easier for a user to do the action related to the widgets when they were closed. We can discuss whenever you want